src/main.c

```diff
@@ -982,6 +983,22 @@ int main_client (int argc, char **argv, GromitData *data)
            else
              data->clientdata = "-1"; /* default to grab all */
          }
+       else if (strcmp (arg, "-l") == 0 ||
```

Comment:
Please also document in the README in the form as the other cmdline options have it.